src/cljs/org/broadinstitute/firecloud_ui/page/workspaces_list.cljs

```diff
@@ -19,15 +19,24 @@
 
 (def row-height-px 56)
 
+(def disabled-text
+  (str "This workspace contains controlled access data. You can only access the contents of this workspace if you are "
+       "dbGaP authorized for TCGA data and have linked your FireCloud account to your eRA Commons account."))
 
 (react/defc StatusCell
   {:render
    (fn [{:keys [props]}]
-     (let [status (get-in props [:data :status])]
-       [:a {:href (nav/create-href (:nav-context props) (get-in props [:data :href]))
+     (let [{:keys [nav-context data]} props
+           {:keys [href status disabled?]} data]
+       [:a {:href (if disabled?
+                    "javascript:;"
+                    (nav/create-href nav-context href))
             :style {:display "block" :backgroundColor (style/color-for-status status)
                     :margin "2px 0 0 2px" :height (- row-height-px 4)
-                    :position "relative"}}
+                    :cursor (when disabled? "default")
+                    :WebkitFilter (when disabled? "grayscale()")
```

Comment:
There are several things that are broken outside of Chrome.  Button disabling has exactly the same problem right now.